app/models/upload.rb

```diff
@@ -40,7 +40,7 @@ def csv_type_check?
   end
 
   def check_for_headers
-    return unless upload_file && csv_type && skip_lines
+    return unless upload_file && csv_type && skip_lines && col_sep
```

Comment:
Chaining `&&` like this makes me want to see a dedicated method call, like `return unless condition_is_met`

Reply:
added method